test/minitest/deprecation_toolkit_plugin_test.rb

```diff
@@ -5,7 +5,11 @@
 
 module Minitest
   class DeprecationToolkitPluginTest < ActiveSupport::TestCase
-    include ActiveSupport::Testing::Isolation
+    if Gem.loaded_specs["activesupport"].version.to_s >= "5.0"
+      include ActiveSupport::Testing::Isolation
+    else
+      parallelize_me!
+    end
```

Comment:
This was the trickiest part to figure out but there must be some fundamental differences b/w `ActiveSupport::Testing::Isolation` because with activesupport 4.2 this code would get a runtime error.